runner/materialize.go

```diff
@@ -151,11 +154,13 @@ func (m MaterializeRunner) Run() (types.CompletionWatcher, error) {
 			EnvVars:  envVars,
 			Image:    WORKER_IMAGE,
 			NumTasks: int32(numChunks),
+			Resource: metadata.ResourceID{Name: m.ID.Name, Variant: m.ID.Variant, Type: provider.ProviderToMetadataResourceType[m.ID.Type]},
 		}
 		kubernetesRunner, err := kubernetes.NewKubernetesRunner(kubernetesConfig)
 		if err != nil {
 			return nil, fmt.Errorf("kubernetes runner: %w", err)
 		}
+		fmt.Printf("Kubernetes config env vars: %v\n", envVars)
```

Comment:
would this print out the environment variables for the pod that executes the pandas script?

Reply:
yes